dropwizard-client/src/test/java/io/dropwizard/client/ssl/DropwizardSSLConnectionSocketFactoryTest.java

```diff
@@ -172,4 +203,13 @@ public void shouldRejectNonSupportedProtocols() throws Exception {
             assertThat(e.getCause()).isInstanceOfAny(SocketException.class, SSLHandshakeException.class);
         }
     }
+    
+    private static class FailVerifier implements HostnameVerifier {
+
+		@Override
+		public boolean verify(String arg0, SSLSession arg1) {
```

Comment:
Could you please replace tabs to spaces? A nitpick, but we prefer to stick to spaces across the project.